src/block/main.css

```diff
@@ -0,0 +1,7 @@
+.main {
+  margin: 0 8px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). In this case, since the `.main__form` class already has a `margin-top`, you should consider removing the margin from the `.main` class or ensure that the vertical margins are only applied in one direction to avoid potential margin collapse issues.